docusaurus/docs/Android/03-ui/04-components/01-channel-list.md

```diff
@@ -4,20 +4,60 @@
 
 ## Overview
 
-<!-- TODO: Brief description and a couple screenshots with default styling. -->
+`ChannelListView` is a component that displays a list of channels available to a user. For users with a slower connection or that don't belong to any channels yet, `ChannelListView` also supports loading and empty states.
+By default, a single list item shows the:
```

Comment:
I broke out the sentence listing these off into an unordered list. Typically, it is good to do that when you are listing more than 3 items or any number of items or options that take more than 2 or 3 words to describe.